src/lib/vocab/policy/permissions.ts

```diff
@@ -1 +1,4 @@
-export const PERMISSIONS: string[] = ['UpdateCommunitySettings', 'DeleteCommunity'];
+export enum PERMISSIONS {
+	UPDATE_COMMUNITY_SETTINGS = 'UpdateCommunitySettings',
+	DELETE_COMMUNITY = 'DeleteCommunity',
+}
```

Comment:
```suggestion
export type PermissionName = keyof typeof permissions;

export const permissions = {
	UpdateCommunitySettings: 'UpdateCommunitySettings',
	DeleteCommunity: 'DeleteCommunity',
};
```

`enum` is something we've avoided so far and I think we should continue -- it's pretty common to do so in the TS world. There's too many caveats and weird things for a little sugar. (the one exception is `@feltcoop/util`'s `LogLevel`, but that's because we leverage the enum integer for the log check, basically a hack) Instead we can use either:

- objects with data (preferred in this case, we're likely to include schema information in the future, not just a string name)
- or a union of strings, e.g. `export type Permission = 'UpdateCommunitySettings' | 'DeleteCommunity'` (which doesn't make sense in this case because we know we'll need additional data with them)

By choosing the first option we can compute the union of strings, which is what `PermissionName` is.